core/contractHost.chainmail

```diff
@@ -0,0 +1,164 @@
+/**
+ * ContractHost provides a way to install and run verifiable contracts.
+ *
+ * Install(source) creates Installations. Installations represent particular
+ * contract forms, which can be spawn()'d to create Contract instances
+ * initialized with specific terms. These Contracts manage invitations for
+ * multiple seats, each of which repesents a role in an interaction.
```

Comment:
Are role and seat the same thing here?

Reply:
Yeah. I am calling the thing a "seat", and explaining the purpose of the thing as a "role". Clearer text would be more welcome.

Reply:
If we use `seat` for the thing, and the purpose of the thing, I think it's a little awkward but the meaning comes through fine. The introduction of `role` really throws me, though, because it sounds like a different thing entirely whose definition I've missed.

Reply:
removed almost all uses of 'role'. I think the remaining ones look clearly like explanation and not like a technical term.